app/views/devise/registrations/edit.html.slim

```diff
@@ -0,0 +1,42 @@
+h2
+  | Edit 
```

Comment:
Are you sure, that there will be ability for registration?

Reply:
yes. user as tenant will possible to register as i understood.